pyproject.client.toml

```
@@ -0,0 +1,135 @@
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[build-system]
+requires = ["setuptools>=78.1.0", "wheel>=0.45.1", "packaging"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "rucio-clients"
+dynamic = ["version"]
+
+# For using SSO login option, install these RPM packages: libxml2-devel xmlsec1-devel xmlsec1-openssl-devel libtool-ltdl-devel
+
+dependencies = [
+        'click',
+        'requests',
+        'urllib3',
+        'dogpile-cache<=1.2.2',
```

```suggestion
        'dogpile-cache',
```

As in https://github.com/rucio/rucio/pull/7672 .

---

Addressed in latest revision